### DISCUSSION TITLE
版面恢复，使用python脚本如何读取pdf？

### DISCUSSION THREAD

**Discussion by greatliu** [PaddlePaddle/PaddleOCR#13664] [Q&A]
### Search before asking

- [X] I have searched the PaddleOCR [Docs](https://paddlepaddle.github.io/PaddleOCR/) and found no similar bug report.

- [X] I have searched the PaddleOCR [Issues](https://github.com/PaddlePaddle/PaddleOCR/issues) and found no similar bug report.

- [X] I have searched the PaddleOCR [Discussions](https://github.com/PaddlePaddle/PaddleOCR/discussions) and found no similar bug report.

### Bug

不好意思，我找来找去只找到python读取图片的版面回复demo，没有读取pdf的啊……pdf的只有命令行说明。

请教怎么读取pdf做版面恢复。

### Environment

python3.9
ppocr 2.8.1

### Minimal Reproducible Example

https://github.com/PaddlePaddle/PaddleOCR/blob/main/ppstructure/docs/quickstart.md#216-%E7%89%88%E9%9D%A2%E6%81%A2%E5%A4%8D
这里只有读取图片的。

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by GreatV** [comment]
参考 https://paddlepaddle.github.io/PaddleOCR/ppstructure/model_train/recovery_to_doc.html#_1

和我们的 cli 接口 https://github.com/PaddlePaddle/PaddleOCR/blob/main/paddleocr.py

**Reply by greatliu** [reply]
> 参考 https://paddlepaddle.github.io/PaddleOCR/ppstructure/model_train/recovery_to_doc.html#_1
> 
> 和我们的 cli 接口 https://github.com/PaddlePaddle/PaddleOCR/blob/main/paddleocr.py

谢谢，我试一下。
